src/edit/dompos.js

```diff
@@ -169,7 +169,7 @@ function scrollIntoView(pm, pos) {
   let coords = coordsAtPos(pm, pos)
   for (let parent = pm.content;; parent = parent.parentNode) {
     let {scrollThreshold, scrollMargin} = pm.options
-    let atBody = parent == document.body
+    let atBody = parent == pm.content.ownerDocument.body
```

Comment:
This will still crash in a shadow DOM, since `atBody` isn't ever going to end up true, and the loop will end up dereferencing null.